save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/SaveConfigReader.kt

```diff
@@ -54,7 +54,7 @@ fun createConfigFromArgs(args: Array<String>): SaveProperties {
     val configFromCli = SaveProperties(args)
     logDebug("Properties after parsed command line args:\n${configFromCli.getFields()}")
     // reading configuration from the properties file
-    val configFromPropertiesFile = readPropertiesFile(configFromCli.propertiesFile ?: defaultConfig().propertiesFile)
+    val configFromPropertiesFile = readPropertiesFile("${(configFromCli.testRootPath ?: defaultConfig().testRootPath)}/save.properties")
```

Comment:
It worked? linux and windows have different dir separators

```suggestion
    val configFromPropertiesFile = readPropertiesFile("${(configFromCli.testRootPath ?: defaultConfig().testRootPath)}${DIRECTORY_SEPARATOR}save.properties")
```

Reply:
Thanks, fixed. I only tested it in mingw, and actually it did work, even though separators in json were `/` instead of `\\`.